GameServerLib/Logic/GameObjects/Champion.cs

```diff
@@ -535,6 +535,10 @@ public override void onCollision(GameObject collider)
 
         public override void DealDamageTo(Unit target, float damage, DamageType type, DamageSource source, bool isCrit)
         {
+            if (HasCrowdControl(CrowdControlType.Disarm))
```

Comment:
Please, this event is called only by AA. Spells dont call this event @Neekhaulas